assets/js/javascript.js

```diff
@@ -197,38 +204,49 @@ $('#rebuildCards').click( function rebuildCards () {
     savedData = JSON.parse(localStorage.getItem('queryArray'));
     recipeCardBuild(savedData);
   } else {
-    console.log('No saved data');
+    console.log('No saved search data');
   }
 });
 //Build cards when called
 function recipeCardBuild (array) {
-  $('.recipeCard').remove();
-    array.forEach ((element,index,array) => {
-      newRecipeCard = $('<div class="recipeCard" name="recipe '+element.id+'"></div>');
-      newRecipeTitle = $('<h3 class="recipeTitle">'+element.title+'</h3>');
-      newRecipeImage = $('<img class="recipeImage" src='+element.image+'>');
-      newRecipeOl = $('<ol class="ingredientList" name="recipe '+element.id+'"></ol>');
-      newRecipeCard.append(newRecipeTitle, newRecipeImage, newRecipeOl);
-      element.usedIngredients.forEach((ele,i,arr) => {
-        newRecipeIngUsed = $('<li class="usedIngredient" aisle="'+ele.aisle+'">'+ele.originalString+'</li>');
-        newRecipeOl.append(newRecipeIngUsed);
-      })
-      element.missedIngredients.forEach((ele2,i2,arr2) => {
-        newRecipeIngMiss = $('<li class="missIngredient" aisle="'+ele2.aisle+'">'+ele2.originalString+'</li>');
-        newRecipeOl.append(newRecipeIngMiss);
+    localStorage.setItem('queryArray', JSON.stringify(array));
+    $('.recipeCard').remove();
+      array.forEach ((element,index,array) => {
+        newRecipeCard = $('<div class="recipeCard" name="recipe '+element.id+'"></div>');
+        newRecipeTitle = $('<h3 class="recipeTitle">'+element.title+'</h3>');
+        newRecipeImage = $('<img class="recipeImage" src='+element.image+'>');
+        newRecipeOl = $('<ol class="ingredientList" name="recipe '+element.id+'"></ol>');
+        newRecipeCard.append(newRecipeTitle, newRecipeImage, newRecipeOl);
+        element.usedIngredients.forEach((ele,i,arr) => {
+          newRecipeIngUsed = $('<li class="usedIngredient" aisle="'+ele.aisle+'">'+ele.originalString+'</li>');
+          newRecipeOl.append(newRecipeIngUsed);
+        })
+        element.missedIngredients.forEach((ele2,i2,arr2) => {
+          newRecipeIngMiss = $('<li class="missIngredient" aisle="'+ele2.aisle+'">'+ele2.originalString+'</li>');
+          newRecipeOl.append(newRecipeIngMiss);
+        })
+        $('#recipeContainer').append(newRecipeCard);
       })
-      $('#recipeContainer').append(newRecipeCard);
-    })
 }
 
 // Added Button Event Listeners
 menuBtn.on('click', printDropMenu)
 mainSrchBtn && mainSrchBtn.on('click', searchAndSave);
 findRecipeBtn && findRecipeBtn.on('click', goToMain)
 
-// Add JOTD to landing page
+// Add JOTD to landing page, check if 24hours has passed since last call
 if (window.location.pathname.endsWith('index.html')) {
-  addJoke();
+  if (localStorage.getItem('day1') == null) { 
+    localStorage.setItem('day1', Date.now());
+    addJoke();
+  } else if (localStorage.getItem('day1') != null) {
```

Comment:
```suggestion
  } else if (localStorage.getItem('day1') != null) {
```
This second If statement seems redundant. Ether something exists in localstorage or it's null.